recipes/flatc/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,4 @@
+int main() 
+{
+	return 0;
```

Comment:
Why don't you consume the generated Vec3 in this file?

Reply:
That would require pulling the correct flatbuffers version as a dependency.  The test_package.cpp is only there for cmake as it otherwise complains.

I could add flatbuffers as a dependency but to me that seemed a bit excessive.